### ISSUE TITLE
Error recognizing BCMath

### ISSUE THREAD

**Issue by DeltaKiloz** [firefly-iii/firefly-iii#935]
Starting a new thread with this since we kept posting on an old closed one from a different issue.

I keep getting HTTP Error 500 after finishing all the steps in the installation guide. Checking the Apache error logs I found the error that seems to be the most consistent:

`PHP Fatal error: Uncaught Error: Call to undefined function bcscale() in /var/www/firefly-iii/bootstrap/app.php:20\nStack trace:\n#0 /var/www/firefly-iii/public/index.php(38): require_once()\n#1 {main}\n thrown in /var/www/firefly-iii/bootstrap/app.php on line 20`

I've verified I have php-bcmath (v7.1) installed. I had also completely uninstalled php7.0 before I upgraded to php7.1 to ensure all the PHP modules that I had to install extra would be consisting with php7.1.

**Comment by JC5** [comment]
Ah good idea. Did you do a reboot in the mean time?

Create a file called `pi.php` and fill it with this:

```
<?php
phpinfo();
```

Then visit `/pi.php` on your server. Does it also say it's PHP 7.1?

**Comment by DeltaKiloz** [comment]
That did the trick!

So I was using a DigitalOcean default LAMP Droplet which already had php7.0 installed, which I uninstalled it and upgraded to php7.1 but apparently everything was still trying to point to php7.0. I went ahead and killed that VPS and built a LAMP from the ground up using only php7.1 and everything lined up perfectly.

One additional self inflicted issue I discovered was that because I had changed the DocumentRoot in `/etc/apache2/sites-enabled/000-default.conf` from ` DocumentRoot /var/www/` to ` DocumentRoot /var/www/firefly-ii/public/` and was browsing to _http://IPaddy/firefly-ii/public/_  would give me an additional 404 error. Correction for this is to either 1) keep the root set ` DocumentRoot /var/www/` and browse to _http://IPaddy/firefly-ii/public/_ (per your installation guide), or 2) to set the root to  ` DocumentRoot /var/www/firefly-ii/public/` and browse to _http://IPaddy/_ . 

Thanks again for all your help, been really looking forward to getting this up and running!
